bin/stashcp

```diff
@@ -97,6 +97,14 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
     payload['start2']=start2
     payload['end2']=end2
     
+    with open(".job.ad") as job_file:
+        for line in job_file.readLines():
+            if "AutoClusterAttrs" in line:
```

Comment:
So, is this kind of what your are looking for?
```Python
    with open(".job.ad") as job_file:
        for line in job_file.readLines():
            if line.lower().find("AutoClusterAttrs") == 0:
                continue
```
Or would you prefer going down the regex route